src/model-summary/index.jsx

```diff
@@ -5,14 +5,17 @@ import size from 'lodash/size';
 import { connect } from 'react-redux';
 import { Snippet } from '../';
 
-const getValue = (value, format, model, nullValue) => {
+const getValue = (value, format, model, nullValue, formatNullValue) => {
   if (!value) {
```

Comment:
I'd just replace this change with a `&& !formatNullValue` in this first conditional, so null values fall through to the existing format call.